docs/troubleshooting-guide/db-connection.md

```diff
@@ -34,6 +34,8 @@ If you don't have access to the Metabase Admin panel, you'll need to ask the per
 
     - If you're running Metabase Cloud, check that you've [whitelisted our IP addresses](https://www.metabase.com/cloud/docs/ip-addresses-to-whitelist).
 
+3. Make sure that Metabase is using a role with the correct privileges to connect to your data warehouse.
```

Comment:
```suggestion
3. Make sure that Metabase is connecting to your database using a database user account that has a role with the privileges to query and write to the tables that you want to query or write to. If the database user account doesn't have a role with those privileges, you'll either need to grant that account a role (or roles) with the necessary privileges, or have Metabase connect using a different database account.
```

Is this more precise? Just want to clarify that, like Metabase's accounts and groups, databases typically have have users and roles.